source/android/mobile/src/main/java/io/adaptivecards/adaptivecardssample/MainActivityAdaptiveCardsSample.java

```diff
@@ -343,38 +348,76 @@ private void renderAdaptiveCard(boolean showErrorToast)
         }
     }
 
+    /**
+     * Render importer UI (defined as an AdaptiveCard) in the importer tab
+     * @param showErrorToast show toast on IO errors, if True
+     */
+    private void renderImporterCard(boolean showErrorToast) {
```

Comment:
>{ [](start = 60, length = 1)

nit: curly braces '{' go in a new line, this document mixes both but let's go with this formatting